boards/arc/hs4xd/hs4xd_defconfig

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_ARC_HS4XD=y
+CONFIG_BOARD_HS4XD=y
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=100
+CONFIG_XIP=y
```

Comment:
Any reason to have XIP enabled?

We don't have any XIP flash memory and we don't have XIP enabled for HSDK v1.